src/phoebus_driver.cpp

```diff
@@ -112,27 +112,46 @@ void PhoebusDriver::PostInitializationCommunication() {
     rad_mocmc_active = (rad->Param<std::string>("method") == "mocmc");
   }
 
+  const int num_partitions = pmesh->DefaultNumPartitions();
+  TaskRegion &sync_region = tc.AddRegion(num_partitions);
+  for (int ib = 0; ib < num_partitions; ib++) {
+    auto &md = pmesh->mesh_data.GetOrAdd("base", ib);
+    auto &tl = sync_region[ib];
+
+    const auto any = parthenon::BoundaryType::any;
+    const auto local = parthenon::BoundaryType::local;
+    const auto nonlocal = parthenon::BoundaryType::nonlocal;
+
+    auto start_recv =
+        tl.AddTask(none, parthenon::cell_centered_bvars::StartReceiveBoundBufs<any>, md);
+
+    auto send = tl.AddTask(start_recv,
+                           parthenon::cell_centered_bvars::SendBoundBufs<nonlocal>, md);
+
+    auto send_local =
+        tl.AddTask(start_recv, parthenon::cell_centered_bvars::SendBoundBufs<local>, md);
+    auto recv_local = tl.AddTask(
+        start_recv, parthenon::cell_centered_bvars::ReceiveBoundBufs<local>, md);
+    auto set_local =
+        tl.AddTask(recv_local, parthenon::cell_centered_bvars::SetBounds<local>, md);
+
+    auto recv = tl.AddTask(
+        start_recv, parthenon::cell_centered_bvars::ReceiveBoundBufs<nonlocal>, md);
+    auto set = tl.AddTask(recv, parthenon::cell_centered_bvars::SetBounds<nonlocal>, md);
+
+    if (pmesh->multilevel) {
+      tl.AddTask(set | set_local,
+                 parthenon::cell_centered_refinement::RestrictPhysicalBounds, md.get());
+    }
```

Comment:
I think if this wasn't originally in the phoebus task list, it should be. This fills the coarse buffers for physical boundary conditions, which is needed. We may not have noticed because we may not have run refinement problems with non-periodic boundaries.